doc/releases/changelog-0.21.0.md

```diff
@@ -19,7 +19,7 @@
   * `qml.sample` or
   * multiple `qml.expval` / `qml.var` measurements.
 
-  Consider a QNode that returns basis state probabilities:
+  Consider a QNode that returns basis-state probabilities:
```

Comment:
For my understanding: would we have `basis-state` here because the phrase describes `probabilities`? We don't seem to have a hyphen in other parts of the code base.

Reply:
Thanks for asking :slightly_smiling_face:. In general, when there is a "compound" modifier (e.g., has multiple words in it), you put a hyphen in. https://www.grammarly.com/blog/hyphen/
Of course, sometimes in English terms become so common that the hyphens get dropped. And in our codebase, we are multiple contributors writing things over many years, so sometimes the standard of the more esoteric English rules are not uniform

Reply:
In this case, you can consider the two alternatives:
- "basis" modifies "state probabilities" (no hypen used); or
- "basis state" modifies "probabilities (then use hyphen)

Reply:
Thank you! :tada: